polkadot/node/core/backing/src/lib.rs

```diff
@@ -1564,7 +1586,17 @@ async fn import_statement<Context>(
 
 	let stmt = primitive_statement_to_table(statement);
 
-	Ok(rp_state.table.import_statement(&rp_state.table_context, stmt))
+	// Don't import statement if the sender is disabled
+	if rp_state.table_context.validator_is_disabled(&stmt.sender) {
+		gum::debug!(
+			target: LOG_TARGET,
+			sender_validator_idx = ?stmt.sender,
+			"Not importing statement because the sender is disabled"
+		);
+		return Ok(None)
```

Comment:
In `sign_import_and_distribute` wouldn't we still distribute a local statement despite having not imported it?

Reply:
Fixed